x-pack/plugins/security_solution/cypress/tasks/api_calls/prepackaged_rules.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { PrePackagedRulesStatusResponse } from '../../../public/detections/containers/detection_engine/rules/types';
+
+export const getPrepackagedRulesStatus = () => {
+  return cy.request<PrePackagedRulesStatusResponse>({
+    method: 'GET',
+    url: 'api/detection_engine/rules/prepackaged/_status',
+    headers: { 'kbn-xsrf': 'cypress-creds' },
+  });
+};
+
+export const getPrepackagedRulesCount = () => {
+  cy.log('Get prepackaged rules count');
+  return getPrepackagedRulesStatus().then(({ body }) => {
+    const prebuiltRulesCount = body.rules_installed + body.rules_not_installed;
+
+    return prebuiltRulesCount;
+  });
+};
```

Comment:
For visibility: we talked that this function should return the number of existing `security-rule` asset saved objects instead of calculating this sum. In order to do this, the `api/detection_engine/rules/prepackaged/_status` endpoint will need to return this value in its response.